docs/modules/iothub.md

```diff
@@ -0,0 +1,61 @@
+# [Azure IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/)
+
+_Note: This module is EXPERIMENTAL. To enable this module, you must run Open Service Broker for Azure with modules.minStability set to `experimental`_
+
+## Services & Plans
+
+### Service: azure-iot-hub
+
+| Plan Name     | Description                                                  |
+| ------------- | ------------------------------------------------------------ |
+| `free`        | IoT hub Free Tier - max 8,000 messages per day.              |
+| `basic-b1`    | IoT hub Basic B1 Tier - max 400,000 messages per day.        |
+| `basic-b2`    | IoT hub Basic B2 Tier - max 6,000,000 messages per day.      |
+| `basic-b3`    | IoT hub Basic B3 Tier - max 300,000,000 messages per day.    |
+| `standard-s1` | IoT hub Standard S1 Tier - max 400,000 messages per day.     |
+| `standard-s2` | IoT hub Standard S2 Tier - max 6,000,000 messages per day.   |
+| `standard-s3` | IoT hub Standard S3 Tier - max 300,000,000 messages per day. |
+
+#### Behaviors
+
+##### Provision
+
+Provisions a new IoT Hub.
+
+###### Provisioning Parameters
+
+| Parameter Name   | Type                | Description                                                  | Required | Default Value                                                |
+| ---------------- | ------------------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ |
+| `location`       | `string`            | The Azure region in which to provision applicable resources. | Y        |                                                              |
+| `resourceGroup`  | `string`            | The (new or existing) resource group with which to associate new resources. | Y        |                                                              |
+| `tags`           | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N        | Tags (even if none are specified) are automatically supplemented with `heritage: open-service-broker-azure`. |
+| `units`          | `int`               | Number of IoT hub units. Each IoT Hub is provisioned with a certain number of units in a specific tier. The tier and number of units determine the maximum daily quota of messages that you can send. **Note**: for plan `free`, this field is invalid; for plan `standard-s3`, allowed values are [1,10]; for other plans, allowed values are [1, 49]. | N        | If not provided, `1 `will be used as default value.          |
```

Comment:
* Add "in the range" previous the expression to ensure users be aware of that it is a expression in math instead of a set?
* The maximum is 50 instead of 49 in the portal?